guacamole-common-js/src/main/webapp/modules/ArrayBufferWriter.js

```diff
@@ -62,6 +62,20 @@ Guacamole.ArrayBufferWriter = function(stream) {
     }
 
     /**
+     * The maximum length of any blob sent by this Guacamole.ArrayBufferWriter,
+     * in bytes. Data sent via
+     * {@link Guacamole.ArrayBufferWriter#sendData|sendData()} which exceeds
```

Comment:
It's the way you provide text for JSDoc3's `@link`: http://usejsdoc.org/tags-inline-link.html

Re-reading the above ... it looks like there's a more readable, more markdown-y syntax available. Perhaps I should switch over to that before precedent is set?

I believe this is our first use of `@link` in the docs.